rllib/algorithms/marwil/tests/test_marwil_rl_module.py

```diff
@@ -0,0 +1,30 @@
+import itertools
+import unittest
+import ray
+
+from pathlib import Path
+
+
+class TestMARWIL(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        ray.init()
+
+    @classmethod
+    def tearDown(self) -> None:
+        ray.shutdown()
+
+    def test_rollouts(self):
+        frameworks = ["tf2"]
```

Comment:
Can we use `torch` here?

Reply:
Oh yes, sure. I have overseen this. Good catch!